docs/api.md

```diff
@@ -51,11 +51,12 @@ If `opts` is specified, then the default options (shown below) will be overridde
 
 ```js
 {
-  dht: Boolean|Object,     // Enable DHT (default=true), or options object for DHT
   maxConns: Number,        // Max number of connections per torrent (default=55)
   nodeId: String|Buffer,   // DHT protocol node ID (default=randomly generated)
   peerId: String|Buffer,   // Wire protocol peer ID (default=randomly generated)
-  tracker: Boolean|Object  // Enable trackers (default=true), or options object for Tracker
+  tracker: Boolean|Object, // Enable trackers (default=true), or options object for Tracker
+  dht: Boolean|Object,     // Enable DHT (default=true), or options object for DHT
+  webSeeds: Boolean        // Enable BEP19 web seeds (default=true)
```

Comment:
This would have been better as singular, `webSeed` IMO since we didn't name the other option `trackers`. No worries, I'll change it along with the other breaking API changes in v1.0.